cmd/containerd/main.go

```diff
@@ -152,24 +154,14 @@ func main() {
 }
 
 func before(context *cli.Context) error {
-	if context.GlobalString("config") == "default" {
-		fh, err := ioutil.TempFile("", "containerd-config.toml.")
-		if err != nil {
-			return err
-		}
-		if _, err := conf.WriteTo(fh); err != nil {
-			fh.Close()
-			return err
-		}
-		fh.Close()
-		log.G(global).Infof("containerd default config written to %q", fh.Name())
-		os.Exit(0)
-	}
 	err := loadConfig(context.GlobalString("config"))
 	if err != nil && !os.IsNotExist(err) {
 		return err
 	} else if err != nil && os.IsNotExist(err) {
 		log.G(global).Infof("config %q does not exist. Creating it.", context.GlobalString("config"))
+		if err := os.MkdirAll(filepath.Dir(context.GlobalString("config")), os.FileMode(0750)); err != nil {
```

Comment:
:thinking: I don't guess so. Leave it completely up to the user.
Will remove the auto-create logic instead.